app/cdap/components/Administration/TetheringTabContent/shared.styles.tsx

```diff
@@ -63,8 +63,6 @@ export const GridHeader = styled.div`
 `;
 
 export const GridBody = styled.div`
-  max-height: 300px;
-  overflow: auto;
```

Comment:
We decided to remove scrolling for now until pagination is done and see if we need scrollable tables.